ru/lessons/basics/enum.md

```diff
@@ -1,25 +1,46 @@
 ---
-version: 0.9.0
+version: 1.2.0
 layout: page
 title: Enum
 category: basics
 order: 3
 lang: ru
 ---
 
-Набор алгоритмов для операций с коллекциями.
+Набор алгоритмов для операций с перечислениями.
 
 {% include toc.html %}
 
 ## Enum
 
-Модуль `Enum` включает в себя более ста функций для работы с коллекциями, которые описаны в предыдущем уроке.
+Модуль `Enum` включает в себя более 70 функций для работы с перечислениями. Все коллекции, о которых мы узнали в [предыдущем уроке](../collections/), за исключением кортежей, являются перечислениями.
 
-Этот урок содержит только небольшую часть функций. Для того, чтобы увидеть полный список функций - посетите официальную документацию по [`Enum`](https://hexdocs.pm/elixir/Enum.html) . Для документации по "ленивым" операциям есть отдельная документация в модуле [`Stream`](https://hexdocs.pm/elixir/Stream.html).
+Этот урок содержит только небольшую часть доступных функций, однако мы можем изучить их самостоятельно.
+Давайте проведем небольшой эксперимент в IEx.
+
+```elixir
+iex> Enum.__info__(:functions) |> Enum.each(fn({function, arity}) ->
+...>   IO.puts "#{function}/#{arity}"
+...> end)
+all?/1
+all?/2
+any?/1
+any?/2
+at/2
+at/3
+...
+```
+
+Становится ясно, что у нас есть огромное количество функций.
+Перечисление лежит в основе функционального программирования и является чрезвычайно полезной вещью.
+Используя его в сочетании с другими особенностями Elixir, такими как документация, которая является объектом первого класса, как мы только что видели, можно невероятно расширить возможности разработчика.
+
+
+Для того, чтобы увидеть полный список функций - посетите официальную документацию по [`Enum`](https://hexdocs.pm/elixir/Enum.html). Для ленивых перечислений воспользуйтесь модулем [`Stream`](https://hexdocs.pm/elixir/Stream.html).
```

Comment:
Sorry for keeping original order. I agree with you, but the sentence still overloaded. In the next translations, I'll try to make a translation more readable.